src/modules/LanItems.rb

```diff
@@ -1382,7 +1382,7 @@ def ip_overview(dev_map)
       if ip =~ /DHCP/
         bullets << format("%s %s", _("IP address assigned using"), ip)
       elsif IP.Check(ip)
-        prefixlen = dev_map["PREFIXLEN"]
+        prefixlen = dev_map["PREFIXLEN"] || @prefix
```

Comment:
This seems wrong: `ip_overview` operates on its argument, but here we reach out for some global state.

This method has no tests. How did the `dev_map` look like for the reported bug? Make a new test with that?

Test coverage shows that the method does get some testing, but the `if` bodies are not covered.